### ISSUE TITLE
This repo can probably be retired

### ISSUE THREAD

**Issue by zeke** [electron-userland/electron-prebuilt-updater#12]
The Electron team has forked this repo to https://github.com/electron/electron-prebuilt-updater and modified it to publish both `electron` and `electron-prebuilt`. The Heroku app we're running now points to that fork, so I think it's safe to retire this repo.

Any objections?

Props to @johnmuhl and @maxogden for creating and maintaining this nice automation setup. We are grateful for your help, and happy to take this burden off you.

@jlord @kevinsawicki @zcbenz

**Comment by johnmuhl** [comment]
Do you mean deleted or something else? I'm not familiar with retiring a repo.

**Comment by zeke** [comment]
> Do you mean deleted or something else?

Yeah I was kind of leaving that open to interpretation. We could delete it entirely (fine with me), or we could add a deprecation note to the readme and the github repo description (also fine with me).

**Comment by johnmuhl** [comment]
I don't see any archival value here. Does deletion have any negative affect on forks?

**Comment by zeke** [comment]
As I discovered last week when npm [closed-sourced their website](http://blog.npmjs.org/post/148119862590/out-with-the-newww-and-in-with-the-www-an-update), the forks live on: https://github.com/zeke/newww

Gonna delete this repo later today if don't hear any objections.
